symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/BotAuthenticatorRsaImpl.java

```diff
@@ -51,10 +51,9 @@ public BotAuthenticatorRsaImpl(
   /**
    * {@inheritDoc}
    */
-  @Nonnull
   @Override
-  public AuthSession authenticateBot() throws AuthUnauthorizedException {
-    AuthSession authSession = new AuthSessionRsaImpl(this);
+  public @Nonnull AuthSession authenticateBot() throws AuthUnauthorizedException {
```

Comment:
As I remember, @symphony-youri told me once that `@Nonnull` should be in front of `public`

Reply:
We need Sonar !